contrib/docker-compose.yml

```diff
@@ -0,0 +1,232 @@
+version: "3.3"
```

Comment:
I think this could be `git cloned` or pulled from the faas repo. `docker service update --image=` will then override the faas-swarm image.